catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/Registry.view.js

```diff
@@ -53,13 +53,14 @@ function (ich,Marionette,_,$,Q,ModalRegistry,EmptyView,wreqr,Utils,deleteRegistr
             _.bindAll(this);
             this.listenTo(this.model, 'change', this.render);
             this.listenTo(wreqr.vent, 'status:update-'+ this.model.attributes.pid, this.updateStatus);
-
+            this.listenTo(wreqr.vent, 'remote:update-'+ this.model.attributes.pid, this.updateRemote);
         },
         serializeData: function(){
```

Comment:
Don't think we need this serialize method. Just change the handlebars to use allowPush and allowPull instead of pushAllowed and pullAllowed.